deepmd/infer/deep_pot.py

```diff
@@ -179,6 +183,8 @@ def eval(
         atomic_virial
             The atomic virial of the system, in shape (nframes, natoms, 9). Only returned
             when atomic is True.
+        hessian
```

Comment:
The returned type annotations in lines 120 and 134 should be updated.

Reply:
`Optional[np.ndarray]` has been added to the tail of returned tuple.